lib/src/base-styles/_nx-lists.scss

```diff
@@ -10,6 +10,7 @@
 
 .nx-list {
   margin: var(--nx-spacing-4x) 0 var(--nx-spacing-6x) 0;
+  overflow-wrap: anywhere;
```

Comment:
Safari still doesn't support this.  It does support `word-break: break-word` which is the more extreme version that will break words in order to fill lines whether it needs to or not.  We probably don't want that by default but I thought it was worth discussing.

Reply:
> Safari still doesn't support this. It does support `word-break: break-word` which is the more extreme version that will break words in order to fill lines whether it needs to or not. We probably don't want that by default but I thought it was worth discussing.

Acknowledged. This has been discussed during WW. Shall proceed with this change knowing that it will still not be fixed in Safari.